backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/DatabaseDocumentNumberRecyclingService.java

```diff
@@ -75,45 +77,110 @@ public Optional<String> addForRecycling(
               .abbreviation(documentationOfficeAbbreviation)
               .build();
 
-      return Optional.of(repository.save(deleted).getDocumentNumber());
+      repository.save(deleted);
 
     } catch (Exception e) {
       log.warn("Won´t reuse the document number", e);
```

Comment:
@Nadav-B : This has not been your task, but in case you're editing the file, we could change this to log.info. don't you think?

Reply:
I hope I understand you correctly, but `addForRecycling` is only called on deletion of documentation unit.

Reply:
Yes, exactly! And when a docunit is deleted and the number can't be recycled, it would count as expected behaviour, right?

Reply:
Oh I see you mean in line 81, I was confused because of the marking. [Done](https://github.com/digitalservicebund/ris-backend-service/commit/208b683f12d7cb7c20bc50431bc3017e0096b90c#diff-ac6fb6e22dffe44d234f1e5390569a0dff92c4a584f7283dba599c9fcd4a55a4)